Automattic/wp-calypso
# Add @automattic/privacy-toolset npm library

#### Proposed Changes

* Introduce a new reusable library for privacy components and tools
* Create a reusable Cookie Banner component optimized for SSR
<img width="1138" alt="CleanShot 2022-11-22 at 10 29 35@2x" src="https://user-images.githubusercontent.com/8419292/203368855-e4f46c67-c720-4527-82f5-7ee536f3e8c3.png">
<img width="1144" alt="CleanShot 2022-11-22 at 10 30 29@2x" src="https://user-images.githubusercontent.com/8419292/203368960-1d0247a2-6c2d-4f07-9d68-7e1f62aef622.png">


#### Testing Instructions

* Checkout the changes
* Run `yarn workspace @automattic/privacy-toolset run storybook`
* Storybook should automatically open in your browser
* Go to the path `?path=/story/cookie-banner--default` (should be open by default)
* The Cookie Banner should be styled as per the design above
* Click "Accept All" - it should result in the action `accept: {essential: true, analytics: true, advertising: true}`
* Click "Customize" - you should see an expanded version of the Cookie Banner as per the design above
* Uncheck the "Analytics" bucket
* Click "Accept Selection" - it should result in the action `accept: {essential: true, analytics: false, advertising: true}`

#### Pre-merge Checklist





Related to #69304
